hadoop-ozone/common/src/main/shellprofile.d/hadoop-ozone-manager.sh

```diff
@@ -19,8 +19,10 @@ if [[ "${HADOOP_SHELL_EXECNAME}" == ozone ]]; then
 fi
 
 _ozone_manager_hadoop_finalize() {
-  if [[ "${HADOOP_CLASSNAME}" == "org.apache.hadoop.ozone.om.OzoneManagerStarter" ]]; then
-   echo "Ozone Manager classpath extended by ${OZONE_MANAGER_CLASSPATH}"
-   hadoop_add_classpath "${OZONE_MANAGER_CLASSPATH}"
+  if [[ "${HADOOP_CLASSNAME}" == "org.apache.hadoop.ozone.om.OzoneManagerStarter" ]] &
```

Comment:
Single `&` starts the command in the background.  Please use `&&` instead.

```suggestion
  if [[ "${HADOOP_CLASSNAME}" == "org.apache.hadoop.ozone.om.OzoneManagerStarter" ]] &&
```

Reply:
Thank @adoroszlai  for the suggest.
Fixed.